app/javascript/orangelight/availability.es6

```diff
@@ -410,4 +415,8 @@ export default class AvailabilityUpdater {
     if (date_string == null) { return ""; }
     return ` - ${date_string}`;
   };
+
+  on_site_use_location(location) {
+    return location == "marquand$stacks";
```

Comment:
@hectorcorrea, since this is only for marquand, at least for now, can we change the name of the method to be on_site_ marquand_use_location

Reply:
Done